string  type
int16   event_id
float64 score